pandas/_libs/tslibs/period.pyx

```diff
@@ -1416,6 +1416,32 @@ cdef class _Period(object):
 
     @property
     def weekday(self):
+        """
+        Get day of the week that a Period falls on.
+
+        Returns
+        -------
+        int
+
+        See Also
+        --------
+        Period.dayofweek : Get the day component of the Period.
+        Period.week : Get the week of the year on the given Period.
+
+        Examples
+        --------
+        >>> p = pd.Period("2018-03-11", "H")
+        >>> p.weekday
+        6
```

Comment:
I think the examples give the idea that a `Period` is a single day. I'd use a Period for a quarter that should make it clearer that this is not a single day.

And also, or here or in the summary, give more details on which day of the week of the 3 months period is being returned.